src/rgw/rgw_op.cc

```diff
@@ -1397,13 +1397,27 @@ void RGWListBuckets::execute()
 
   uint64_t max_buckets = s->cct->_conf->rgw_list_buckets_max_chunk;
 
+  rgw_user *accounts_user;
+  rgw_user buckets_user;
+
+  if (doing_swift_cross_tenant) {
+	buckets_user.tenant = buckets_user.id = s->bucket_tenant;
```

Comment:
nit: bad formating

Reply:
Ah.  I think you mean the "ldout" lines next, not these.  I have a personal convention of leaving temporary code unindented.  I will remove the ldout's unless others see any value to them.